GeneralsMD/Code/Tools/wdump/chunk_d.cpp

```diff
@@ -1183,6 +1247,24 @@ void	ChunkTableClass::List_W3D_CHUNK_SCG(ChunkItem * Item,CListCtrl *List)
 		data++;
 	}
 }
+
+void	ChunkTableClass::List_W3D_CHUNK_FXSHADER_IDS(ChunkItem* Item, CListCtrl* List)
+{
+	int Counter = 0;
+	int counter = 0;
```

Comment:
These 2 counters with almost identical name are intended?

Reply:
This is wild. I wonder if that is a bug. Counter is always 0 and every `AddItem` call will take a 0.

Reply:
Very sneaky. Ok then all good.